sys/fuchsia/fidlgen/main.go

```diff
@@ -69,6 +76,41 @@ func main() {
 		if err != nil {
 			failf("fidlgen failed: %v\n", err)
 		}
+
+		newFiles = append(newFiles, fmt.Sprintf("%s.txt", txtPath))
+	}
+
+	var unused []ast.Pos
+	desc := ast.ParseGlob("*.txt", nil)
+	if desc != nil {
+		unused = compiler.CollectUnused(desc)
+	}
+
+	for _, file := range newFiles {
+		st, err := os.Stat(file)
```

Comment:
Use osutil.WriteFile instead of ioutil.WriteFile. It uses some common permissions that we use when write any other files. Then we don't need os.Stat at all.

Reply:
Done.